connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockActionTranslator.java

```diff
@@ -106,9 +108,27 @@ public void translate(PlayerActionPacket packet, GeyserSession session) {
                 // Handled in BedrockInventoryTransactionTranslator
                 break;
             case START_BREAK:
-                ClientPlayerActionPacket startBreakingPacket = new ClientPlayerActionPacket(PlayerAction.START_DIGGING, new Position(packet.getBlockPosition().getX(),
-                        packet.getBlockPosition().getY(), packet.getBlockPosition().getZ()), BlockFace.values()[packet.getFace()]);
-                session.sendDownstreamPacket(startBreakingPacket);
+                Matcher m = null;
```

Comment:
Regex isn't necessary for something like this. You just need to check if the identifier is fire with a simple `.equals` check.

Reply:
Fire changes its ID based on age in my testing.

Reply:
I might be missing something though

Reply:
No, Redned is saying that you don't need to use Regex to check it. Just use the `.contains()` or `.equals()` function that is a String function.

Reply:
Ok, thanks for clearing that up